dali/pipeline/workspace/workspace.h

```diff
@@ -189,7 +190,7 @@ class WorkspaceBase : public ArgumentWorkspace {
   /**
    * Set requested batch size for all outputs
    */
-  void set_batch_size(int batch_size) {
+  void SetBatchSizes(int batch_size) {
```

Comment:
```suggestion
  void SetBatchSize(int batch_size) {
```
You are still setting one batch size, to all outputs

Reply:
My way of thinking here is, that it is in fact single value of batch size, but it sets it for number of outputs. Possibly in future there might be more value**s** of batch sizes, e.g. one per every output.

After all, that's what this function does:
```c++
    batch_sizes_.clear();
    batch_sizes_.resize(NumOutput(), batch_size);
```
fills `batch_sizes_` container with (multiple) value**s**